src/main/java/io/vertx/core/eventbus/impl/EventBusImpl.java

```diff
@@ -261,6 +261,9 @@ public EventBus unregisterDefaultCodec(Class clazz) {
   @Override
   public void close(Handler<AsyncResult<Void>> completionHandler) {
     unregisterAllHandlers();
+    if (metrics != null) {
+      metrics.close();
```

Comment:
to which particular context should the EventBus run ? that's why I haven't done anything here.